options/create.go

```diff
@@ -137,6 +139,26 @@ func (opts *Create) Hash() hash.Hash {
 	return hash
 }
 
+func (opts *Create) resolveRemote(env []string) {
+	if opts.RemoteInfo == nil {
+		return
+	}
+
+	var remoteCmd string
+
+	if opts.WorkingDirectory != "" {
```

Comment:
Currently Create.Resolve() will set the same working directory for both the local and remote command. Is this what we want or should they be separate?

Reply:
I think that's fine. I think that if you're in a remote manager than you should be able to set the local directory just the same as you can locally.

Reply:
This will cause the ssh command to fail unless both the local filesystem and the remote one have this directory (e.g. if you want to use `/home/user` as the remote working directory, `/home/user` will have to exist on the local machine as well).

Also, if we're going to keep this overloaded `WorkingDirectory`, you have to modify `Create.Validate()` to not stat the WorkingDirectory in the remote case.